internal/ignition/ignition.go

```diff
@@ -125,6 +125,12 @@ allow chronyd_t container_file_t:sock_file write;
 allow chronyd_t spc_t:unix_dgram_socket sendto;
 `
 
+const agentFixBZ1964591 = `#!/usr/bin/sh
+
+IMAGE=$(echo $1 | sed 's/:.*//')
+podman images | grep $IMAGE || podman rmi --force $1 || true
```

Comment:
Could you write a small comment here saying why we are doing this? And also a go comment